NAS2D/Renderer/Fade.cpp

```diff
@@ -30,15 +30,13 @@ SignalSource<>& Fade::fadeComplete()
 }
 
 
-// Fade in from fadeColor
 void Fade::fadeIn(unsigned int durationInMilliseconds)
 {
 	setDuration(durationInMilliseconds);
 	mDirection = FadeDirection::In;
 }
 
 
-// Fade out to fadeColor
```

Comment:
I guess technically the `fadeColor` is maybe not totally obvious from looking at just this one function. That might be documentation worth keeping.

Reply:
It was not obvious that comment had anything to do with the code written. Nowhere in the function name, signature, or body is `color ` ever used. Hence, the comment got removed.

Reply:
Indeed, though the fade does involve a color. Such as fade to white or fade to black. It's because it's not anywhere in the function that it needs to be documented. It's the `fadeColor` method that sets that aspect of fading.

Reply:
That's a different function. Long-form documentation can (should) be written to relate the two (docs.microsoft does this all the time). Trying to do so in the comments is just adding noise.

Reply:
I feel like this was never really addressed.

Reply:
I was in the middle of a small refactor that combined `fadeColor` and `fadeIn/Out` into `fade`:

```cpp
void fadeIn(Color fadeColor, std::chrono::milliseconds fadeInTime) {
//...
}
void fadeOut(Color fadeColor, std::chrono::milliseconds fadeOutTime) {
//...
}
```

Reply:
That's not a documentation change though. If you're making such a change on another branch, then you should probably avoiding changing the documentation here preemptively.